README.md

```diff
@@ -110,4 +112,63 @@ NSUInteger
 CGFloat
 BOOL
 ```
-Most other variables you create will be objects. The next reading will discuss string objects and string formatting in more detail.
+Most other variables you create will be objects.
+
+## Code Commenting
+
+Comments are text that the compiler is told to ignore. They're useful for inserting notes to yourself or to other programmers about the code. There a couple of different syntaxes that you might see:
+
+```objc
+// A line comment is declared with two slashes and comments out
+// all the text to the right of the slashes.
+```
+
+```objc
+/*
+A block comment will comment-out all the text between 
+its opening and closing symbols. They use a slash and star 
+to open, and a star and slash to close.
+*/
+```
+
+Commenting has its usefulness but in general should be treated as a last resort. Learn to keep your commenting to a minimum and use the code itself to document your intent.
```

Comment:
I'd go a little less strongly on this. "Commenting is useful for documenting non-obvious behavior, but it is not a substitute for well-written code. Learn to keep your commenting..."